pkg/compiler/test/testdata/branch/k8s/mycereal_condition_has_no_deps.json

```diff
@@ -6,7 +6,10 @@
     "namespace": "namespace",
     "creationTimestamp": null,
     "labels": {
+      "domain": "domain",
       "execution-id": "name",
+      "project": "hello",
+      "shard": "6",
```

Comment:
this is not really shard right? Shard is logical, it is more like a bin? Or I guess we should crisply define the semantics

Reply:
how does "shard-key" sound? i'm having trouble liking shard-bin, it sounds like a collection of shards - when really we're defining shards by a collection of these values.